### PR TITLE
chore: auto-publish specification

### PR DESCRIPTION
**Pull request by marcoscaceres**
security-privacy-questionnaire.md now [lives in the wiki](https://github.com/w3c/push-api/wiki/Security-and-Privacy-Questionnaire-for-the-Push-API).

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/push-api/pull/331.html" title="Last updated on Jun 16, 2021, 1:37 AM UTC (8e427e0)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/push-api/331/35457b2...8e427e0.html" title="Last updated on Jun 16, 2021, 1:37 AM UTC (8e427e0)">Diff</a>

### CODE REVIEW

**Comment by marcoscaceres** [comment]
Depends on https://github.com/w3c/push-api/pull/330

File: index.html [hunk 1]
```diff
@@ -455,11 +455,11 @@ <h2>
...
         </h2>
         <figure>
-          <a href="sequence_diagram.png"><img src="sequence_diagram.png" width="795" height="870"
-          alt=
+          <a href="images/sequence_diagram.png"><img src="images/sequence_diagram.png" width="795"
+          height="870" alt=
           "Example flow of events for subscription, push message delivery, and unsubscription"></a>
           <figcaption>
-            Example flow of events for subscription, push message delivery, and unsubscription
+            Example flow of events for subscription, push message delivery, and unsubscription.
```
Comment (martinthomson) [inline index.html]: Captions are fragments and so usually don't get a period.

File: security-privacy-questionnaire.md [hunk 2]
```diff
@@ -1,99 +0,0 @@
-# Security and Privacy Questionnaire for the Push API
```
Comment (martinthomson) [inline security-privacy-questionnaire.md]: Not sure whether this is a necessary part of this.  If it's dead (it certainly looks that way), I would have expected this to be in a separate changeset.  So I just want to make sure that this is deliberate.

**Review by martinthomson — APPROVED** [review]
Thanks @marcoscaceres, good to keep things updated.

File: security-privacy-questionnaire.md (hunk 2 above)
Comment (marcoscaceres) [reply security-privacy-questionnaire.md]: Yeah, I was being lazy. I'll resend. It's not dead - just moved to the wiki https://github.com/w3c/push-api/wiki/Security-and-Privacy-Questionnaire-for-the-Push-API

File: index.html (hunk 1 above)
Comment (marcoscaceres) [inline index.html]:
```suggestion
            Example flow of events for subscription, push message delivery, and unsubscription
```

**Comment by martinthomson** [comment]
@marcoscaceres it might pay to rebase this one before we merge it.  Can you do that?
